ngraph/frontend/paddlepaddle/include/paddlepaddle_frontend/place.hpp

```diff
@@ -17,47 +17,171 @@
 #pragma once
 
 #include <frontend_manager/frontend_manager.hpp>
+// #include "framework.pb.h"
 
 namespace ngraph {
-namespace frontend {
-
-class PlacePDPD : public Place
-{
-    // TODO
-};
-
-class InPortPlacePDPD : public PlacePDPD
-{
-    // TODO
-};
-
-class OutPortPlacePDPD : public PlacePDPD
-{
-    // TODO
-};
-
-class VarPlacePDPD;
-class OpPlacePDPD : public PlacePDPD
-{
-public:
-    InputModel* model;
-    const void* op; // TODO: make it cleaner
-    std::map<std::string, std::vector<std::weak_ptr<VarPlacePDPD>>> outputs;
-    std::map<std::string, std::vector<std::weak_ptr<VarPlacePDPD>>> inputs;
-    OpPlacePDPD(const void* _op) : op(_op) {}
-};
-
-class VarPlacePDPD : public PlacePDPD
-{
-public:
-    InputModel* model;
-    const void* var; // TODO: make it cleaner
-    std::vector<std::weak_ptr<OpPlacePDPD>> producing_ops; // should never have more than 1 element
-    std::vector<std::weak_ptr<OpPlacePDPD>> consuming_ops;
-    PartialShape shape;
-    element::Type type;
-    VarPlacePDPD(const void* _var) : var(_var) {}
-};
-
-} // namespace frontend
+    namespace frontend {
+
+        class PDPDPlace : public Place {
```

Comment:
It is more consistent to leave it as PlacePDPD not PDPDPlace

Reply:
done